lib/mayaUsd/ufe/ProxyShapeHierarchy.cpp

```diff
@@ -133,6 +133,8 @@ const UsdPrim& ProxyShapeHierarchy::getUsdRootPrim() const
 
 Ufe::SceneItem::Ptr ProxyShapeHierarchy::sceneItem() const { return fItem; }
 
+#if defined(UFE_V4_FEATURES_AVAILABLE) && (UFE_MINOR_VERSION > 4 || UFE_PATCH_LEVEL > 3)
```

Comment:
This should just use a single test like this:
```suggestion
#if (UFE_PREVIEW_VERSION_NUM >= 4004)
```